src/components/base/Text.tsx

```diff
@@ -60,11 +62,17 @@ const Text: React.FC<TextProps> = ({
     className,
   );
 
+  let optionalProps = {};
+  if (id !== '') {
+    //@ts-ignore
+    optionalProps['id'] = id;
+  }
+
   if (area) {
-    return <p className={`${classes} h-full`}>{children}</p>;
+    return <p {...optionalProps} className={`${classes} h-full`}>{children}</p>;
```

Comment:
Please let me know if there is a better way with React for optional empty props. With `id={id}` I saw empty ids on elements which I do not want.